README.md

```diff
@@ -20,6 +20,17 @@ npm install
 npm start
 ```
 
+# Development
```

Comment:
There're three ways (not three steps) to open the devtools window: command line, config file, or shortcut key.